### ISSUE TITLE
'.selectFile must only be invoked from the spec file or support file' error in Firefox

### ISSUE THREAD

**Issue by RKNeedsHelp** [cypress-io/cypress#27845]
### Current behavior

I use this code to upload a File which works perfectly in Chrome, Edge and Electron but not in Firefox:
`    cy.delayedGet('[type="file"]').selectFile('Test Word bestand.pdf', { force: true });
`

I have also tried this version which results in the same behavior:
`    cy.get('input[type=file]').selectFile('cypress/fixtures/Test Word bestand.pdf', { force: true, action: 'drag-drop' });
`

The errror:
`cy.selectFile() must only be invoked from the spec file or support file.`

### Desired behavior

For the file to successfully upload also in Firefox

### Test code to reproduce
```js
    cy.get('input[type=file]').selectFile('cypress/fixtures/Test Word bestand.pdf', { force: true, action: 'drag-drop' });

    cy.delayedGet('[type="file"]').selectFile('Test Word bestand.pdf', { force: true });
```

### Cypress Version

v13.2.0

### Node version

v18.17.1

### Operating System

Windows 22H2

### Other

 Issue is similar to https://github.com/cypress-io/cypress/issues/27149 and #27457

**Comment by nagash77** [comment]
Hi @RKNeedsHelp 
In order to give our engineers the best chance at recreating your problem, please create a reproducible example using a fork of [Cypress Test Tiny](https://github.com/cypress-io/cypress-test-tiny).  This gives us the best chance of seeing exactly what you are seeing and being able to investigate effectively.

**Comment by KrzysztofWisniewskiAcc** [comment]
This should not be closed as it exists on 13.3.2

node:   18.18.2
npm:    10.2.0
cypres: 13.3.2
system: win11

I have only cypress installed.
Runing e2e specs tests

```
describe('=== Some spec ===', function () {
    context('+++ Test file +++', function () {
        it('- Put file into input', {}, function () {
            cy.get('#file-input-xxx').selectFile('cypress/fixtures/files/test.csv');
        })
    })
})
```
